### ISSUE TITLE
MeshLayer and transitions

### ISSUE THREAD

**Issue by grahambates-sp** [uber/deck.gl#2604]
It would appear that the MeshLayer does not currently support transitions (have tried for getPosition, and can see transition flag is not set in the code).

Will this eventually be supported?

**Comment by Pessimistress** [comment]
Yes that's the plan. The layer is experimental so its APIs are not quite aligned with the core layers. If it's blocking you you're welcome to submit a PR (it should just work by adding the `transition: true` flag).

**Comment by grahambates-sp** [comment]
ok great.

FYI : I did try adding the field to my own version of Mesh Layer but it didn't work.

**Comment by charlieforward9** [comment]
https://github.com/user-attachments/assets/cc77cfb2-85b9-4fa8-b526-983664251715

> it should just work by adding the transition: true flag

https://github.com/visgl/deck.gl/blob/6b265fd810df7704b9842136752029e554e98f0a/modules/mesh-layers/src/simple-mesh-layer/simple-mesh-layer.ts#L253

Looks good to close
